packages/go_router/pubspec.yaml

```
@@ -1,7 +1,7 @@
 name: go_router
 description: A declarative router for Flutter based on Navigation 2 supporting
   deep linking, data-driven routes and more
-version: 14.6.0
+version: 14.7.0
```

this should be 14.6.1 since there isn't new api
```suggestion
version: 14.6.1
```